autoparams/src/main/java/org/javaunit/autoparams/BigIntegerGenerator.java

```diff
@@ -0,0 +1,15 @@
+package org.javaunit.autoparams;
+
+import java.math.BigInteger;
+import java.util.Optional;
+
+final class BigIntegerGenerator implements ObjectGenerator {
+    @Override
+    public Optional<Object> generate(ObjectQuery query, ObjectGenerationContext context) {
+        return query.getType().equals(BigInteger.class) ? factory() : Optional.empty();
+    }
+
+    private Optional<Object> factory() {
+        return Optional.of(new BigInteger(64, RANDOM));
```

Comment:
```suggestion
        int maxBitLength = 64;
        return Optional.of(new BigInteger(maxBitLength, RANDOM));
```

사소한 의견입니다만 이렇게 변수 이름을 주면 이해하기 좋을 것 같습니다.

-----------------------

64 값은 `64 bits = 8 bytes = Long`  형 타입 길이로 이해됩니다. `BigInteger` 에 걸맞게 256 정도는 어떨지 의견드립니다.

Reply:
네 그냥 적당한 값을 넣었는데, Long 을 넘어서는 값으로 수정하겠습니다.

Reply:
maxBitLength 변수 추가하였습니다~